state/protocol/util.go

```diff
@@ -69,3 +70,17 @@ func IsSporkRootSnapshot(snapshot Snapshot) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsEpochCommitted returns whether the epoch is committed.
```

Comment:
Lets be explicit with documentation:
```suggestion
// IsEpochCommitted returns whether the epoch is committed.
// No errors are expected during normal operations. 
```